media/js/firefox/welcome/welcome10-experiment.js

```diff
@@ -32,6 +32,9 @@
             cop.init();
         }
     };
-    initTrafficCop();
+
+    if (href.indexOf('automation=true') {
```

Comment:
Oh wait, this should be:
```suggestion
    if (href.indexOf('automation=true') === -1) {
```

Reply:
Doh, I was just thinking that. We only want to initTrafficCop when NOT `automation=true`